week2/homework/src/index.js

```diff
@@ -1,3 +1,61 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+let fs = require('fs');
+let command = process.argv[2];
+let item = process.argv[3];
+switch(command){
+    case 'add' :
+        add(item);
+        break
+    case 'remove':
+        remove(item);
+        break;
+    case 'list':
+        list(item);
+        break;
+    case 'reset':
+        reset(item);
+        break;
+    case 'help' :
+        console.log( `Usage:
+        - Each command is supposed to be preceded by "node . "
+        - you have the following commands
+           -- add        Adds a to-do item into .data.txt. Each item is entered inside " ".
+           -- remove     Removes a to-do item according to its index number.
+           -- reset      Removes all to-do items from the list.
+           -- list       Shows current to-dos, or shows an appropriate text if there are no to-dos
+           -- help       Show  information of cli commands`);
+           break;
+};
+function add (item){
+    fs.appendFile('./data.txt', item + '\n', (error)=>{
+        if (error){
+            console.log(error);
+        };
+    });
+};
+function remove (item){
+    let arrayRemove= fs.readFileSync('./data.txt').toString().split('\n');
+    arrayRemove.splice(item,1);
```

Comment:
Just a note here: you are using javascript's 0-based index.
As a user of your app I don't know that and I expect to provide a 1-based index.

Consider this list:
```
todo 1
todo 2
todo 3
```

`node . remove 1` will remove `todo 2`, do you think this is the expected behavior?

Again, the  function works fine, this is a UX issue :)